RevenueCatUI/Templates/TemplateViewType.swift

```diff
@@ -161,7 +161,11 @@ extension View {
 
     func adaptTemplateView(with configuration: TemplateViewConfiguration) -> some View {
         self
+        #if os(visionOS)
+            .background(Material.regular)
```

Comment:
We might want to do this below on `backgroundContent` so we only use this if blurred background is enabled. Otherwise it'd ignore paywalls configured with a color and the foreground text might become ineligible.